llvm/test/Feature/Repo/repo_hash_contributions_func.ll

```diff
@@ -0,0 +1,55 @@
+; This test is used to check the hash calculation for the contributions.
+;
+; All GOs' information are shown below:
+; GO Name: Var1, Initial Dependencies: [ Func1 ], Contributions: [ ]
+; GO Name: Var2, Initial Dependencies: [ ], Contributions: [ ]
+; GO Name: Func1, Initial Dependencies: [ ], Contributions: [ ]
+; GO Name: Main, Initial Dependencies: [ Func2 ], Contributions: [ Var2 ]
+; GO Name: Func2, Initial Dependencies: [ ], Contributions: [ Var1 ]
+;
+; This test checks that `Var2` digest is changed when `Func1` is modified.
+;
+; The test invokes five steps:
+;  1) Compile the initial code `Inputs/repo_contributions_dependencies_loop.ll` to %t.
+;  2) Compile `repo_hash_contributions.ll` to %t1.
+;  3) Dump the digest of `Var2` in %t.
+;  4) Dump the digest of `Var2` in %t1.
+;  5) Check the digests of `Var2` are different since the function `Func1` is changed.
+;
+; RUN: rm -rf %t.db
+; RUN: env REPOFILE=%t.db clang -c -O0 --target=x86_64-pc-linux-gnu-repo -x ir %S/Inputs/repo_contributions_dependencies_loop.ll -o %t
```

Comment:
The LLVM Testing Infrastructure Guide [recommends](https://www.llvm.org/docs/TestingGuide.html#extra-files) that ”If your test requires extra files besides the file containing the RUN: lines and the extra files are small, consider specifying them in the same file and using split-file to extract them.” Even after trimming as much as possible, our files are probably still too large for this to be practical, but it might be worth a try.

Reply:
Nice to know this. However, look like this feature is introduced in LLVM 12. Our branch is updated to LLVM 11 which doesn't have the `split-file` project in LLVM tools.